ptf/tests/common/base_test.py

```diff
@@ -117,13 +120,10 @@ def get_controller_packet_metadata(p4info, meta_type, name):
                         return m
 
 
-def de_canonicalize_bytes(bitwidth: int, input: bytes):
+def de_canonical(bitwidth: int, input: bytes):
```

Comment:
Let's call this for what it is :)
```suggestion
def pad(bitwidth: int, input: bytes):
```